docs/integrations/monitoring/datadog.md

```diff
@@ -2,31 +2,115 @@
 
 You can add operational visibility to your [Rack](../../reference/primitives/rack) with Datadog.
 
+## Configure Kubectl to Point at Your Rack
```

Comment:
kubectl and kubeconfig should be lowercase and `kubectl` should be backticked anywhere it appears outside of titles